src/bootstrap/test.rs

```diff
@@ -1592,7 +1594,9 @@ note: if you're sure you want to do this, please open an issue as to why. In the
 
             // Only pass correct values for these flags for the `run-make` suite as it
             // requires that a C++ compiler was configured which isn't always the case.
-            if !builder.config.dry_run() && matches!(suite, "run-make" | "run-make-fulldeps") {
+            if !builder.config.dry_run()
```

Comment:
I was reluctant to touch this comment because it seems to already be somewhat out-of-date (the C++ compiler parts have been moved elsewhere), and I‘m not quite sure what to replace it with. But I suppose leaving it as-is isn't great either.